build-n-publish.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/sh
+
+pkgName="doppler-menu-mfe"
```

Comment:
Related to https://github.com/FromDoppler/doppler-user-sites-integration-mfe/pull/2/files#r854369688
```suggestion
pkgName="doppler-user-sites-integration-mfe"
```